tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -123,12 +122,14 @@ class VarDictJavaEndToEnd(tumorBam: PathToBam,
                           maxThreads: Int = 32) extends Pipeline {
   import VarDictJava.BinDir
 
+  // Put these here so that they'll error at construction if missing
+  private val biasScript = BinDir.resolve("teststrandbias.R")
+  private val vcfScript  = BinDir.resolve("var2vcf_valid.pl")
+
   def build(): Unit = {
     val tn = tumorName.getOrElse(VarDictJava.extractSampleName(bam=tumorBam))
     val dict = PathUtil.replaceExtension(ref, ".dict")
-
-    val biasScript = BinDir.resolve("teststrandbias.R")
-    val vcfScript  = BinDir.resolve("var2vcf_valid.pl")
+    val tmpVcf = Io.makeTempFile("vardict.", ".vcf", dir= if (out == Io.StdOut) None else Some(out.getParent))
```

Comment:
This is a little hokey, but without having the constructor take in a new directory arg it's not clear what would be better.